webhooks.md

```diff
@@ -503,19 +503,21 @@ curl -X POST \
   }'
 ```
 
-`transaction.reimbursement.status` event is triggered when a reimbursement changes state on a transaction.
+`reimbursement.pending`, `reimbursement.issued`, `reimbursement.failed` events are triggered when a reimbursement changes state. The `reimbursement.pending` when the new state is `pending`, `reimbursement.issued` when a new state is `issued`, and `reimbursement.failed` when a new state is `failed`
```

Comment:
Should we maybe document it like this?
"The `reimbursement.pending` when the reimbursement is created"
"and `reimbursement.failed` when it fails to complete the reimbursement"

Purely as example.

Reply:
The lifecycle of the reimbursement is already explained in the document, and the webhook names are clearly mapped to statuses, it's clear enough in my opinion